### ISSUE TITLE
ngen config layer support

### ISSUE THREAD

**Issue by hellkite500** [NOAA-OWP/ngen-cal#73]
Need to add layer information/support to ngen config.

This spec isn't well defined, and is undergoing a bit of refactoring even now, but we need to add it soon.

**Comment by aaraney** [comment]
@hellkite500, is there an issue on NextGen or a design doc floating around somewhere that can be linked here?

**Comment by hellkite500** [comment]
Not really, I'm in the midst of refactoring and trying to better formalize the layer stuff.

**Comment by aaraney** [comment]
Just keep me in the loop here, @hellkite500 and Ill make any required changes.

**Comment by hellkite500** [comment]
Layer configuration for ngen: https://github.com/NOAA-OWP/ngen/blob/master/include/realizations/config/layer.hpp
